perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -362,6 +374,10 @@ sub inspect : Private {
         }
 
         if ($valid) {
+            if ( $reputation_change != 0 && !$problem->get_extra_metadata('user_reputation_set') ) {
+                $problem->user->update_reputation($reputation_change);
+                $problem->set_extra_metadata( user_reputation_set => 1)
```

Comment:
Hmm, do you mean that if a report is unhidden the reputation decrement should be undone? I think in the interests of keeping things simple each report should only ever have one effect on the user's reputation. Open to suggestions otherwise though.